### PR TITLE
TNPG: send pushes in batches.

### PR DESCRIPTION
**Pull request by aforge**

### CODE REVIEW

File: server/push/tnpg/push_tnpg.go [hunk 1]
```diff
@@ -26,6 +26,7 @@ type configType struct {
 	Enabled          bool `json:"enabled"`
 	Buffer           int  `json:"buffer"`
 	CompressPayloads bool `json:"compress_payloads"`
+	BatchSize        int  `json:"batch_size"`
```
Comment (or-else) [inline server/push/tnpg/push_tnpg.go]: FCM SendAll supports batches of 100 messages and that's it:
https://godoc.org/firebase.google.com/go/messaging#Client.SendAll

Configuring more than 100 won't work. Configuring fewer is just wasteful. I see no point in letting users configure this value if FCM does not let us configure it. A hardcoded 100 seems like the best choice.

Comment (aforge) [reply server/push/tnpg/push_tnpg.go]: Sure.

File: server/push/tnpg/push_tnpg.go [hunk 2]
```diff
@@ -23,10 +23,11 @@ type Handler struct {
...
 
 type configType struct {
-	Enabled          bool `json:"enabled"`
-	Buffer           int  `json:"buffer"`
-	CompressPayloads bool `json:"compress_payloads"`
-	BatchSize        int  `json:"batch_size"`
+	Enabled          bool   `json:"enabled"`
+	Buffer           int    `json:"buffer"`
+	CompressPayloads bool   `json:"compress_payloads"`
+	BatchSize        int    `json:"batch_size"`
```
Comment (or-else) [inline server/push/tnpg/push_tnpg.go]: As I wrote in the other PR I would make this hardcoded 100.

Comment (aforge) [reply server/push/tnpg/push_tnpg.go]: This is an outdated snapshot. I removed it.

**Pull request merged**